herddb-docker/src/main/kubernetes/samples/bookkeeper/4_bookkeeper.deployment.yaml

```diff
@@ -0,0 +1,27 @@
+## from https://raw.githubusercontent.com/apache/bookkeeper/master/deploy/kubernetes/gke/bookkeeper.yaml
+
+##
+## Run BookKeeper auto-recovery from a different set of containers
+## Auto-Recovery makes sure to restore the replication factor when any bookie
+## crashes and it's not recovering on its own.
+##
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: bookie-autorecovery
+spec:
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: bookkeeper
+        component: bookkeeper-replication
+    spec:
+      containers:
+        - name: replication-worker
+          image: apache/bookkeeper:latest
+          command: [ "/bin/bash", "/opt/bookkeeper/entrypoint.sh" ]
+          args: ["/opt/bookkeeper/bin/bookkeeper", "autorecovery"]
```

Comment:
this file is not needed for demo purposes,
this is a daemon that replicates data in case of lost bookies

if we want to make things simple you can drop this

Reply:
The templates are not for demo purposes but aims at giving a skeleton for real deployments IMHO.
For demo it is ok to use a plain ServerMain, no?

Reply:
You are right, it should be better to have it.

We are not using autorecovery in production, as we usually have a small set of bookies and we are running autorecovery only manually and when needed (like when you lose definitively a machine)

so okay to keep this file now
thanks